docs/participate/run-nodes/full-node.mdx

```diff
@@ -152,18 +88,15 @@ sudo adduser <username>
 sudo usermod -aG sudo <username>
 ```
 
-3. Log into the server as a new user using ssh
+3. Log into the new user
 
 ```bash
-ssh <username>@<server-ip-address>
+su - username
 ```
 
```

Comment:
Not sure but I think you need to log in to the server with new user and not just switch it, since mytonctrl uses `logname` instead of `whoami` if parent is root: https://github.com/ton-blockchain/mytonctrl/blob/966f1a71030be1f0f86aeb578f0eaa0f4826f3ea/scripts/install.sh#L103.
Can you double check it?